search-ui/src/interfaces/corp-type.ts

```diff
@@ -0,0 +1,403 @@
+/**
+ * Corp type codes, used as keys to get corp info.
+ * (sorted by code)
+ */
+ export enum CorpTypeCd {
```

Comment:
can you move these enums into the enum folder and then import them back into here? I know it doesn't really matter that much, but I think it makes it a little nicer to read / follow